_data/Shivaansh/shivaansh.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Shivaansh Agarwal
+institution: GLA University, Mathura (India)
+profile_pic: shivaansh.jpg
+quote: "Whether you think you can, or you can't you're right..."
```

Comment:
Please follow the given [instruction](https://github.com/education/graduation#3-add-your-profile-information) by removing the quotes to maintain the same format

Reply:
Removed the quotes from the quote.